src/components/simulationParameters/useFormState.ts

```diff
@@ -95,7 +96,11 @@ const useFormState = (tasksFromModel: AllModelTasks, gateways: Gateways, eventsF
                                 to: yup.string().required(REQUIRED_ERROR_MSG),
                                 beginTime: yup.string().timeFormat(INVALID_TIME_FORMAT),
                                 endTime: yup.string().timeFormat(INVALID_TIME_FORMAT),
-                                probability: yup.number().typeError(SHOULD_BE_NUMBER_MSG).min(0, SHOULD_BE_LESS_OR_EQ_1_MSG).max(1, SHOULD_BE_GREATER_0_MSG).optional()
+                                probability: yup.number().when('model_type', {
+                                        is: ModelType.FUZZY,
+                                        then: yup.number().required(REQUIRED_ERROR_MSG),
+                                        otherwise: yup.mixed().strip(),
+                                }),
```

Comment:
Did you test this change?
We validate json against this schema when we start simulation (e.g., click `Start simulation` button).
At the moment, I receive the following error: 
<img width="222" alt="image" src="https://github.com/AutomatedProcessImprovement/prosimos-frontend/assets/14131790/2b4ec0f7-66ea-48f9-aaf3-eb29dabf4a6d">